PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -554,11 +554,11 @@ def miniAOD_customizeCommon(process):
 
     # L1 pre-firing weights for 2016 and 2017
     from Configuration.Eras.Modifier_run2_L1prefiring_cff import run2_L1prefiring
-    from Configuration.Eras.Modifier_stage1L1Trigger_cff import stage1L1Trigger
+    from Configuration.Eras.Modifier_stage2L1Trigger_cff import stage2L1Trigger
     from Configuration.Eras.Modifier_stage2L1Trigger_2017_cff import stage2L1Trigger_2017
-    process.load("PhysicsTools.PatUtils.L1ECALPrefiringWeightProducer_cff")
-    stage1L1Trigger.toModify(process.prefiringweight, DataEra = "2016BtoH")
-    stage2L1Trigger_2017.toModify(process.prefiringweight, DataEra = "2017BtoF")
+    process.load("PhysicsTools.PatUtils.L1PrefiringWeightProducer_cff")
+    stage2L1Trigger.toModify(process.prefiringweight, DataEraECAL = "2017BtoF", JetMaxMuonFraction = -1, DoMuons = cms.bool(False) )
```

Comment:
Is the era correct here ("2017BtoF")?

Reply:
if we want to reproduce the configuration previous to this PR to comply with the rule forbidding changes to RECO, yes. (these 2017 maps were also mistakenly picked previously for 2016 ...)

Reply:
> if we want to reproduce the configuration previous to this PR to comply with the rule forbidding changes to RECO, yes. (these 2017 maps were also mistakenly picked previously for 2016 ...)

Ok, thank you!